controllers/domain/getDomain.js

```diff
@@ -106,7 +110,7 @@ module.exports = async (req, res) => {
         });
       }
     }
-
+    console.log(':::data', data);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging entire data objects directly to the console can potentially expose sensitive information and also clutter the log output, making it harder to find relevant information. It's generally a good practice to remove such logs before committing the code. If you need to keep this log for debugging purposes, consider logging only the necessary properties or use a proper logging library that supports different log levels and formats.

```diff
-    console.log(':::data', data);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
Please remove console.log